monkey/monkey_island/cc/ui/src/components/map/MapPageWrapper.tsx

```diff
@@ -17,7 +17,7 @@ const MapPageWrapper = (props) => {
 
   function getPropagationEvents() {
     let url_args = {'type': 'PropagationEvent', 'success': true};
-    return IslandHttpClient.get(APIEndpoint.agentEvents, url_args)
+    return IslandHttpClient.get(APIEndpoint.agentEvents, url_args, {}, true)
```

Comment:
Can you confirm this only gets called by the user? It never gets called automatically?

Reply:
If you leave your Map page on then it does get called.

Reply:
Then this should be `false`